### ISSUE TITLE
Components don't indicate disability outside color in Design.

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#787]
**Describe the bug**
Components don't indicate disability outside color in Design. They are displayed on screen but are not functional.

**To Reproduce**
1. Go to Design (http://localhost:9992/cm/app/?view=widgetbuilder&site=test&path=%2FSites%2Ftest%2F&pathType=page)
2. Create a new site and then create a new page with "Image Auto List" widget.
3. Click on the page and then go to Layout Tab.
4. Click on Action Menu. The Edit Page option in Disabled but it don't indicate disabled outside color.

**Expected behavior**
Disability should be indicated
